Source/Demos/Demo.PrimitiveBatch/Game1.cs

```diff
@@ -0,0 +1,102 @@
+﻿using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Microsoft.Xna.Framework.Input;
+using MonoGame.Extended.Graphics;
+using MonoGame.Extended.Graphics.Batching;
+using SpriteBatch = MonoGame.Extended.Graphics.Batching.SpriteBatch;
```

Comment:
I leave it in your hands to rename it. I don't think we can leave it called `SpriteBatch` though, that's sure to confuse someone.